### ISSUE TITLE
Unicode characters printing issue on all drivers less in the FakeDriver.

### ISSUE THREAD

**Issue by BDisp** [gui-cs/Terminal.Gui#2103]
On `Windows Terminal`:
Pay attention at the spaces before and after the `é`.

![imagem](https://user-images.githubusercontent.com/13117724/196726962-55e760b8-88b3-4a6e-9d1e-e93234316e7d.png)

On Windows host and WSL:
Pay attention at the space after the `é`.

![imagem](https://user-images.githubusercontent.com/13117724/196726314-388575f5-ce53-43b2-b0fa-b7b0dd397d53.png)

On Ubuntu installed in the `VirtualBox` is working well:

![imagem](https://user-images.githubusercontent.com/13117724/196754873-2ba3606a-7edd-413f-91ec-d079b197e530.png)

The FakeDriver is printing well:
```cs
┌──────────────┐    
│Les Misérables│    
◄              └───┐
│hi2               │
└──────────────────┘
```

So, I think it's a Windows 11 bug that is causing this. Hope that someone from `Microsoft` see this.

**Comment by tig** [comment]
Same font/typeface everywhere?

**Comment by BDisp** [comment]
> Same font/typeface everywhere?

On the first two yes. Was the MesloLGS NF. On the VirtualBox I think was the Monospace font.

**Comment by tig** [comment]
@BDisp can you submit a PR that updates the `TabViews` Scenario to illustrate this?

**Comment by BDisp** [comment]
> @BDisp can you submit a PR that updates the `TabViews` Scenario to illustrate this?

But is already there, navigate to the right until you reach that tab.

**Comment by tig** [comment]
> > @BDisp can you submit a PR that updates the `TabViews` Scenario to illustrate this?
> 
> But is already there, navigate to the right until you reach that tab.

Sorry. Missed that:

<img width="451" alt="image" src="https://user-images.githubusercontent.com/585482/200036856-bf464ef9-36a9-4c09-8362-a7f555646de5.png">

**Comment by BDisp** [comment]
Ups I'm seeing the cursor on the tab and on mine image it isn't showing.

**Comment by tig** [comment]
> Ups I'm seeing the cursor on the tab and on mine image it isn't showing.

Yep, see #2175

**Comment by tig** [comment]
@BDisp what's the status of this Issue?

**Comment by BDisp** [comment]
> @BDisp what's the status of this Issue?

Stagnant for now but not forgotten.

**Comment by tig** [comment]
I believe this is addressed with #2932?

**Comment by BDisp** [comment]
> I believe this is addressed with #2932?

No it isn't @tig. This issue was already fixed on the v1 before the v2 started. I maybe forgot to refer this issue when I fixed. So I'm closing this, because it was related with to runes that can be normalized and #2932 is for combining marks that can't be normalized.
